docs/developer/api-reference/enums/allocation-strategy-enum.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: allocation-strategy-enum
+title: AllocationStrategyEnum
+hide_table_of_contents: false
+---
+
+Determine the allocation strategy for the channel.
+
+    PRIORITIZE_SORTING_ORDER - the allocation is prioritized by the warehouses' sort
```

Comment:
Maybe reuse sentence from the guide?
`allocate stocks according to the warehouses' order within the channel`

Reply:
Same here. I will update the schema first.